modules/installation-osp-converting-ignition-resources.adoc

```diff
@@ -41,9 +41,7 @@ files.append(
     'mode': 420,
     'contents': {
         'source': 'data:text/plain;charset=utf-8;base64,' + hostname_b64,
-        'verification': {}
```

Comment:
Just making sure that it's okay for the previous item to have a comma after it, even once this is removed. Same for below and a few other places.

Reply:
@wjiangjay can you confirm the comma at the end of line 43 above? This was taken from the [OSP dev UPI](https://github.com/openshift/installer/blob/master/docs/user/openstack/install_upi.md#edit-the-bootstrap-ignition) docs, but it seems it would no longer be necessary. Similar question [here](https://github.com/openshift/openshift-docs/pull/24805/files#diff-3bbaa3d7bf8e605abcef9d3320cfb62cR58) and [here](https://github.com/openshift/openshift-docs/pull/24805/files#diff-3bbaa3d7bf8e605abcef9d3320cfb62cR121).

Reply:
@jianlinliu @wjiangjay can you confirm our suspicions on the erroneous commas in this OSP script?

Reply:
Yeah, the ending comma at line 43 and line 45 should be removed. Here a online json/yaml converter - https://www.json2yaml.com/, that can help you confirm format issue.

Reply:
Thanks for that resource, and confirmations!